modules/vnet_peering/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "name_prefix" {
+  description = "Prefix name appended to the peering names."
+  default     = ""
+  type        = string
+}
+
+variable "local_peer_config" {
+  description = <<-EOF
+  A map that contains the local peer configuration.
+  Mandatory Values: 
```

Comment:
since [this](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/blob/82370696752ed913e697945e114e4d0830fe4fa6/examples/dedicated_vmseries_and_autoscale/variables.tf#L258) format is more or less approved, can you align variables to match it? It will be less work when we will refactor the other modules.